### PR TITLE
Refactor Kotlin styles

### PR DESCRIPTION
**Pull request by gizemb**
## What's new in this PR?

- Moved button & text styles to different folders
- Refactored naming for more concise inheritance
- Button style now extends from Widget.AppCompat.Button instead of a TextView style

#### APK
[Download build #1139](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1139/artifact/build/artifact/wire-dev-PR2582-1139.apk)

### CODE REVIEW

File: app/src/main/res-v2/values/styles_button.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Button" parent="@style/Widget.AppCompat.Button">
+        <item name="android:fontFamily">sans-serif-light</item>
```
Comment (mejdoo) [inline app/src/main/res-v2/values/styles_button.xml]: I don't think all the buttons in the app will have Roboto Light as font family. Probably we need to refactor again later...

Comment (BradleyWire) [reply app/src/main/res-v2/values/styles_button.xml]: I hope most of the buttons will have the same font to promote consistency, if, some button don't use sans-serif-light then we can override that in its own style?

Comment (mejdoo) [reply app/src/main/res-v2/values/styles_button.xml]: Same think for `textAllCaps false`. For the new welcome screen buttons, it should be `textAllCaps true`

**Comment by mejdoo** [comment]
build again

**Comment by mejdoo** [comment]
I wil merge this to develop

**Comment by mejdoo** [comment]
@BradleyWire please rebase your work to develop

**Pull request merged**